lib/src/extensions/int_array.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Extension methods to convert integer arrays to List<int>
+
+import 'dart:ffi';
+
+extension Uint8Helper on Pointer<Uint8> {
+  /// Creates a [List] from `Pointer<Uint8>`.
```

Comment:
Probably worth explaining in the comment why these methods are necessary. That is, of course, these Lists are a copy of the original list that is backed by Dart memory, versus `.asTypedList(length)`, which lives on the C heap and may not remain available.

Reply:
These methods and also these https://github.com/timsneath/win32/blob/b68ef9fbedbada9dcaa7b6994c856001acfa9457/lib/src/extensions/comobject_pointer.dart#L29 https://github.com/timsneath/win32/blob/b68ef9fbedbada9dcaa7b6994c856001acfa9457/lib/src/extensions/hstring_array.dart#L25 are used in VectorHelper's `toList` methods to create a Dart `List` from the pointer filled by the `getMany` methods. 

It is highly unlikely the consumers will need these helpers as they should use `toList()` methods provided by `IVector` and `IVectorView`.

Because of that, I've been thinking maybe we shouldn't expose these extension methods to consumers and move these into the `winrt/internal` folder in order to prevent any confusion that might arise.

Reply:
Yeah, that makes sense. The fewer extension methods exposed on common intrinsic types like `int` or `String`, the better, I think. We should still explain the difference between this and `.asTypedList` though, for someone reading the code who might not immediately recognize why we're copying the list.

Reply:
Great! I pushed some new commits. Can you take another look?